QuickOSM/core/parser/preset_parser.py

```diff
@@ -0,0 +1,90 @@
+""" parser of the default presets file """
+
+import xml.dom.minidom as xml
+
+from QuickOSM.qgis_plugin_tools.tools.resources import resources_path
+
+PRESET_PATH = resources_path('JOSM_preset', 'defaultpresets.xml')
+
+__copyright__ = 'Copyright 2021, 3Liz'
+__license__ = 'GPL version 3'
+__email__ = 'info@3liz.org'
+
+
+class PresetsParser:
+
+    def __init__(self):
+
+        self.file = xml.parse(PRESET_PATH)
+
+    def parser(self):
+
+        def fetch_keys(group, heirs_name: list):
```

Comment:
Same, docstrings and annotations for the return.
I'm trying to understand what `heirs` mean ?